openedx/features/course_experience/views/welcome_message.py

```diff
@@ -28,7 +28,7 @@ def render_to_fragment(self, request, course_id=None, **kwargs):
         Returns: A fragment, or None if there is no welcome message.
         """
         course_key = CourseKey.from_string(course_id)
-        course = get_course_with_access(request.user, 'load', course_key, check_if_enrolled=True)
+        course = get_course_with_access(request.user, 'load', course_key, check_if_enrolled=False)
```

Comment:
If (for now) we are saying that we won't display the welcome message unless the user is enrolled, why is this changing?